docs/isp_flashing_guide.md

```diff
@@ -129,4 +199,45 @@ You should see a couple of progress bars, then you should see:
 
 Which means everything should be ok! Your board may restart automatically, otherwise, unplug your Teensy and plug in your keyboard - you can leave your Teensy wired to your keyboard while testing things, but it's recommended that you desolder it/remove the wiring once you're sure everything works.
 
+If you're using a SparkFun PocketAVR Programmer, or another USB Tiny based ISP programmer, you will want to use something like this: 
+
+    avrdude -c usbbiny -P usb -p atmega32u4
+
+#### Advanced: Changing Fuses
+
+If you're switching bootloaders, such as flashing QMK DFU on a Pro Micro, you will need to change the fuses, in additional to flashing the bootloader hex file.  This is because `caterina` (the Pro Micro bootloader) and `dfu` handle the startup routines differently, and that behavior is controlled by the fuses.  
+
+!> This is one area that it is very important to be careful, as changing fuses is one of the ways that you can perminently brick your controller.  
+
+For this, we are assuming the 5V 16MHz versions of the `atmega32u4` (such as the 5V Pro Micro).
+
+For DFU on the `atmega32u4`, these are the fuse settings that you want: 
+
+|----------|------------------|
+| Low      | `0x5E`           |
+| High     | `0xD9` or `0x99` |
+| Extended | `0xC3`           |
+
+The High fuse can be 0xD9 or 0x99. The difference is that 0xD9 disables JTAG, which QMK Firmware disables via software as well, while 0x99 doesn't disable JTAG. 
+
+To set this add `-U lfuse:w:0x5E:m -U hfuse:w:0xD9:m -U efuse:w:0xC3:m` to your command.  So the final command should look something like: 
+
+    avrdude -c avrisp -P COM3 -p atmega32u4 -U flash:w:main.hex:i -U lfuse:w:0x5E:m -U hfuse:w:0xD9:m -U efuse:w:0xC3:m
+
+For Caterina on the `atmega32u4`, these are the fuse settings that you want: 
+
+|----------|--------|
+| Low      | `0xFF` |
+| High     | `0xD9` |
+| Extended | `0xC3` |
+
+To set this add `-U lfuse:w:0xFF:m -U hfuse:w:0xD8:m -U efuse:w:0xC3:m` to your command.  So the final command should look something like: 
+
+    avrdude -c avrisp -P COM3 -p atmega32u4 -U flash:w:main.hex:i -U lfuse:w:0xFF:m -U hfuse:w:0xD8:m -U efuse:w:0xC3:m
+
+
+If you are using a different controller or want different configuration, you can use [this AVR Fuse Calculator](http://www.engbedded.com/fusecalc/) to find a better value for you.
```

Comment:
This fuse calculator doesn't seem to give the same values for 32u4 DFU as you gave in the table above, but [this one does](http://eleccelerator.com/fusecalc/fusecalc.php).

Please advise.

Reply:
Default has Extended as 0xF3, but seems to actually default to 0xC3, which is why I picked that value.  

I believe that the defaults that the link uses is what docs say.  However, IIRC, the actual difference matter in practice.

Reply:
> * Note that some numerical values refer to fuses containing undefined bits (set to '1' here). Depending on the target device these fuse bits will be read either as '0' or '1'. Verification errors will occur if the values are read back with undefined bits set to '0'. Everything is fine if the values read from the device are either the same as programmed, or the following values (undefined set to '0'): Extended: 0x03.

Reply:
@fauxpark So, would the recommended be `0x03` for the extended fuse, then?  Or?

Reply:
I just tried setting the efuse to F3 and 03, either way avrdude complains it read back C3. So that is probably the "true" default.